### ISSUE TITLE
No patients 

### ISSUE THREAD

**Issue by Cas-tani** [alanwoolley/CorsixTH-Android#136]
Hi! First of all thank you for what you are doing. Unfortunately I don't have any patients in my hospital. I bought a game at EA games, and copied a folder to my phone. I don't have any HOSP folder anywhere. The issue happened both on beta and in a normal, non-beta game. In a game I have a reception, a GP office, personnel and has opened the hospital. It should work.
Please help me.

**Comment by JamesPoel** [comment]
Did you hire a receptionist? Without one, patients will not turn up...

**Comment by Cas-tani** [comment]
Yes, I did. I am playing the game on the PC, and it all goes well, patients
come, are cured, etc.

**Comment by alanwoolley** [comment]
A change in 0.68.0 should improve this. https://github.com/CorsixTH/CorsixTH/pull/2497

I hope to get this integrated over the next few weeks.

**Comment by FalloutClassicGamer** [comment]
> A change in 0.68.0 should improve this. https://github.com/CorsixTH/CorsixTH/pull/2497
> 
> I hope to get this integrated over the next few weeks.
> 

Bro please update cortixth android
